cg/meta/upload/scout/balsamic_config_builder.py

```diff
@@ -48,14 +48,17 @@ def build_config_sample(self, db_sample: models.FamilySample) -> ScoutBalsamicIn
         )
         if analysis_type == "tgs":
             analysis_type = "panel"
+        if analysis_type == "wgs":
+            raise SyntaxError("Not allowed to upload WGS cancer cases")
```

Comment:
Do we maybe wanna we use a custom CgError for this?

Reply:
I thought it was ok since it will be removed anyday